admin/resources/js/components/poolCandidate/UpdatePoolCandidate.tsx

```diff
@@ -0,0 +1,310 @@
+            />
+            <MultiSelect
+              id="acceptedOperationalRequirements.sync"
+              name="acceptedOperationalRequirements"
+              label="Operational Requirements: "
+              placeholder="Select one or more operational requirements..."
+              options={operationalRequirementOptions}
+              rules={{ required: errorMessages.required }}
+            />
+            <MultiSelect
+              id="expectedSalary"
+              label="Expected Salary: "
+              name="expectedSalary"
+              placeholder="Select one or more expected salaries..."
+              options={enumToOptions(SalaryRange).map(({ value }) => ({
+                value,
+                label: getSalaryRange(value),
+              }))}
+              rules={{ required: errorMessages.required }}
+            />
+            <MultiSelect
+              id="expectedClassifications"
+              label="Expected Classifications: "
+              placeholder="Select one or more classifications..."
+              name="expectedClassifications"
+              options={classificationOptions}
+              rules={{ required: errorMessages.required }}
+            />
+            <MultiSelect
+              id="cmoAssets"
+              label="Cmo Assets: "
+              name="cmoAssets"
+              options={cmoAssetOptions}
+              rules={{ required: errorMessages.required }}
+            />
+            <Select
+              id="status"
+              label="Status: "
+              name="status"
+              rules={{ required: errorMessages.required }}
+              options={[
+                { value: "", label: "Select a status..." },
+                ...enumToOptions(PoolCandidateStatus),
+              ]}
+            />
+            <Submit />
+          </form>
+        </FormProvider>
+      </section>
+    );
+  };
+
+export const UpdatePoolCandidate: React.FunctionComponent<{
+  poolCandidateId: string;
+}> = ({ poolCandidateId }) => {
+  const [poolCandidateState] = useGetPoolCandidateQuery({
+    variables: { id: poolCandidateId },
+  });
+  const { fetching: fetchingPoolCandidate, error: poolCandidateError } =
+    poolCandidateState;
+  const [lookupResult] = useGetPoolCandidatesForFormQuery();
+  const {
+    data: lookupData,
+    fetching: fetchingLookupData,
+    error: lookupDataError,
+  } = lookupResult;
+  const classifications: Classification[] | [] =
+    lookupData?.classifications.filter(notEmpty) ?? [];
+  const cmoAssets: CmoAsset[] = lookupData?.cmoAssets.filter(notEmpty) ?? [];
+  const operationalRequirements: OperationalRequirement[] =
+    lookupData?.operationalRequirements.filter(notEmpty) ?? [];
+
+  const [_result, executeMutation] = useUpdatePoolCandidateMutation();
+  const handleUpdatePoolCandidate = (
+    id: string,
+    data: UpdatePoolCandidateInput,
+  ) =>
+    executeMutation({ id, poolCandidate: data }).then((result) => {
```

Comment:
We should use lodash/pick to ensure `data` only has the fields we need. See UpdateUser.tsx for a similar example:
```
/* We must pick only the fields belonging to UpdateUserInput, because its possible
       the data object contains other props at runtime, and this will cause the
       graphql operation to fail. */
```